test/pkg/environment/common/expectations.go

```diff
@@ -696,6 +696,19 @@ func (env *Environment) EventuallyExpectDrifted(nodeClaims ...*corev1beta1.NodeC
 	}).Should(Succeed())
 }
 
+func (env *Environment) ConsistentlyExpectNodeClaimsNotDrifted(duration time.Duration, nodeClaims ...*corev1beta1.NodeClaim) {
+	GinkgoHelper()
+	for _, nc := range nodeClaims {
+		By(fmt.Sprintf("consistently expect nodeclaim %s not to be drifted for %s", nc.Name, duration))
```

Comment:
Rather than printing multiple times in a loop, can we just print once for all of the nodeClaims so that this doesn't get spammy with a lot of NodeClaims.